packages/data-point/lib/resolve/reducer-path.js

```diff
@@ -19,6 +19,15 @@ function resolveObjectPath (acc, jsonPath) {
     return _.get(acc, jsonPath.slice(2))
   }
 
+  if (jsonPath.slice(-2) === '[]') {
+    // take last entry in json path without []
+    // $a.b.c[] => `$a.b` and `c`
```

Comment:
This is probably a mistake on my part of not explaining the feature correctly, the idea of the map is to map the full path, not the last key. 

This means if we have object: 

```js
[
  {a: {b: c: 'foo'}},
  {a: {b: c: 'bar'}}
]
```

the Reducer path should look like: `$a.b.c[]`

I'm sorry for the confusion

Reply:
No problem. I think I have it right now.